### PR TITLE
created khan-droid.json

### PR DESCRIPTION
**Pull request by khan-droid**
## Proposed Changes

  -Created a file khan-droid.json to add my profile to the link tree

### CODE REVIEW

**Review by voidz7 — APPROVED** [review]
Looks good to me ⚡

**Comment by khan-droid** [comment]
> Looks good to me ⚡

Thank you

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

**Review by MukulKolpe — APPROVED** [review]

**Review by aayush89890 — APPROVED** [review]
Welcome to the community.

**Review by FayasNoushad — APPROVED** [review]
Also add a profile picture on your github profile.

**Comment by voidz7** [comment]
> Also add a profile picture on your github profile.

Tell them this is a suggestion we cannot force someone to do so right ? 🦅

**Comment by FayasNoushad** [comment]
> > Also add a profile picture on your github profile.
> 
> Tell them this is a suggestion we cannot force someone to do so right ?

Yeah, Just my suggestion.

File: public/data/khan-droid.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "http://github.com/khan-droid",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/khalid_khan111",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (eddiejaoude) [inline public/data/khan-droid.json:17]: Needs a blank line at the end, see the red mark by GitHub on line 17

File: public/data/khan-droid.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Khalid Khan",
+    "bio": "Upcoming Full stack developer | Open source contributor",
+    "avatar": "https://github.com/khan-droid.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/khan-droid",
```
Comment (Panquesito7) [inline public/data/khan-droid.json:8]:
```suggestion
        "url": "https://github.com/khan-droid",
```

File: public/data/khan-droid.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/khalid_khan111",
```
Comment (Panquesito7) [inline public/data/khan-droid.json:13]:
```suggestion
        "url": "https://twitter.com/khalid_khan111",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/khan-droid.json (hunk 1 above)
Comment (Panquesito7) [reply public/data/khan-droid.json:17]:
```suggestion
}

```

**Comment by eddiejaoude** [comment]
I will merge and fix the formatting

**Pull request merged**